elements/asq-java-q-presenter.html

```diff
@@ -3,6 +3,14 @@
 <link rel="import" href="../../iron-list/iron-list.html">
```

Comment:
this element should use the  `ASQ.asqQuestionElementBehavior` like [here]( https://github.com/ASQ-USI-Elements/asq-multi-choice-q/blob/master/elements/asq-multi-choice-q-presenter.html#L121) . The same holds true for the viewer element